include/oneapi/dpl/experimental/kt/internal/esimd_radix_sort_dispatchers.h

```diff
@@ -372,7 +371,7 @@ __onesweep(sycl::queue __q, _RngPack1&& __pack, _RngPack2&& __pack_out, ::std::s
 template <bool __is_ascending, ::std::uint8_t __radix_bits, bool __in_place, typename _RngPack1, typename _RngPack2,
           typename _KernelParam>
 sycl::event
-__radix_sort(sycl::queue __q, _RngPack1&& __pack_in, _RngPack2&& __pack_out, _KernelParam __param)
+__radix_sort(sycl::queue __q, _RngPack1&& __pack_in, _RngPack2&& __pack_out, _KernelParam)
```

Comment:
May we stay this type `_KernelParam` only in template parameters?

Reply:
I am inclined towards keeping it as is.
The same approach is in another KT - single pass scan. Also this signature matches the public interface.

Reply:
Up to you.
But I think that something that required parameter created in run-time but used only type info from it - is a little bit strange pattern.

Reply:
Yes, it is not a straightforward approach.

I'd like to keep it to avoid cascade effect of  `__param` being unused in upper layers. 
Also, this run-time variable should be optimized out, see a minimal example https://godbolt.org/z/rv4azYxf9 (foo and bar have equivalent assembly).

Reply:
It's not optimization problem. It's design problem: you are forcing the user of the interface to create instances of types that are simply not needed at run time.

Reply:
Thanks, that's a good point to stress in an RFC for this KT, which I am going to create...